roda-app.rb

```diff
@@ -45,9 +45,9 @@ class RodaApp < Roda
     # /
     r.root do
       env['warden'].authenticate!
-
       response['Content-Type'] = 'text/html'
-      @reservations = Reservation.today
+      r.params['day'] ? @day = ( Date.parse r.params['day'] ) : @day = Date.today
```

Comment:
Line is too long. [81/80]<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `Date.today` without zone. Use `Time.zone.today` instead.